test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 freezegun
 odoo_test_helper
+odoo14-addon-stock_move_source_relocate @git+https://github.com/OCA/wms@refs/pull/290/head#subdirectory=setup/stock_move_source_relocate
```

Comment:
please don't hide this into the migration commit. Add an extra commit like `Add TMP dependency` which makes prominent there's an extra commit to be discarded :wink: